tests/verifyDBaaSUpgrade_test.js

```diff
@@ -49,7 +69,7 @@ const pxcDbClusterDetails = new DataTable(['namespace', 'clusterName', 'node']);
 
 pxcDbClusterDetails.add(['default', `${pxc_cluster_name}`, '0']);
 
-Data(pxcDbClusterDetails).Scenario('PMM-T726 Verify PXC cluster monitoring after PMM Server upgrade @upgrade-dbaas',
+Data(pxcDbClusterDetails).Scenario('PMM-T726 Verify PXC cluster monitoring after PMM Server upgrade @upgrade-dbaas-after',
```

Comment:
Just a suggestion maybe better flag name would be @post-dbaas-upgrade. To be consistent.

Reply:
All tags go upgrade-dbaas-xxx seems pretty consistent to me.